core-lib/TestSuite/Minitest.ns

```diff
@@ -441,12 +441,12 @@ OTHER DEALINGS IN THE SOFTWARE. *)
               exClass:: (ClassMirror reflecting: e) classObject.
               exClass == TestFailureException ifTrue: [
                 promisePair resolve: (testContextInstance
-                  ifNil:    [ TestFailure case: self description: ex messageText]
-                  ifNotNil: [:it | it createFailureResultFor: self description: ex messageText])
+                  ifNil:    [ TestFailure case: self description: e messageText ]
```

Comment:
Giving that the surrounding code uses `ex` as name for the exception variable, and `exClass` for the name of the class, wouldn't it be better to stay consistent with that naming?